packages/atlas/src/components/_navigation/SidenavViewer/FollowedChannels.styles.ts

```diff
@@ -17,31 +17,51 @@ export const ChannelsTitle = styled(Text)`
   margin-bottom: ${sizes(4)};
   padding-left: ${NAVBAR_LEFT_PADDING}px;
   width: ${EXPANDED_SIDENAVBAR_WIDTH - NAVBAR_LEFT_PADDING}px;
-  color: ${oldColors.gray[300]};
+  color: ${cVar('colorText')};
+  text-transform: uppercase;
+  letter-spacing: 0.07em;
```

Comment:
All these props should already come from the `Text` component (`color` will come from `isSecondary` prop), don't think those are needed here